api/yelp_beans/routes/tasks.py

```diff
@@ -104,3 +108,40 @@ def clean_user_subscriptions():
             logging.info(f"Deleted UserSubscriptionPreference<{preference.id}>")
 
     return "OK"
+
+
+@tasks.route("/generate_meeting_requests_for_auto_opt_in_preferences", methods=["GET"])
+def generate_meeting_requests_for_auto_opt_in_preferences():
+    specs = get_specs_for_current_week()
+
+    logging.info("All specs: ")
+    logging.info(specs)
+
+    for spec in specs:
+        spec_timezone = timezone(spec.meeting_subscription.timezone)
+        spec_dt = spec.datetime.replace(tzinfo=utc).astimezone(spec_timezone)
```

Comment:
Should we be using the `get_meeting_datetime` function [here](https://github.com/MinaAtYelp/beans/blob/da41ded9234f8f408516c9dfd1841241fa6760da/api/yelp_beans/logic/meeting_spec.py#L65)?
```suggestion
       spec_dt = get_meeting_datetime(spec)
```